tools/cldr-apps/src/main/webapp/js/esm/cldrMenu.js

```diff
@@ -112,7 +112,7 @@ function loadInitialMenusFromJson(json, theLocale) {
 
   setupCoverageLevels(json, theLocale);
 
-  cldrLoad.continueInitializing(json.canAutoImport);
+  cldrLoad.continueInitializing(json.canAutoImport || false);
```

Comment:
leftover from debugging?

Reply:
no, actually I noticed that json.canAutoImport may be undefined, in which case I'd rather pass false than undefined, since a boolean is expected -- though it wouldn't really matter in this case